case-lib/lib.sh

```diff
@@ -343,3 +343,19 @@ disable_kernel_check_point()
 {
     KERNEL_CHECKPOINT="disabled"
 }
+
+is_sof_logger_disabled()
+{
+    # get the firmware type
+    manifest=/etc/sof/manifest.txt
+    FIRMWARE_TYPE=$(grep "firmwaretype" < "$manifest" | awk -F ':' '{print $2}' | sed -e 's/^[ ]*//g;s/\"//g')
```

Comment:
No need for the `<`.

No need for the `grep`, `awk` can already do `grep` (and many other things): `awk -F ':' '/firmwaretype/ {print $2 }'`

Regular expressions with only special characters are time-consuming to read and deserve a comment, for instance `# remove whitespace`.